File: core/src/backendServer/models/BouncerProjectile.js
const Projectile = require('./Projectile');
const ProjectileType = require('./ProjectileType');
const constants = require('../settings/constants');

class BouncerProjectile extends Projectile {
    constructor(x = 0, y = 0, xSpeed = 0, ySpeed = 0, id, agentId) {
        super(x, y, xSpeed, ySpeed, constants.BOUNCER_PROJECTILE_WIDTH / 2, ProjectileType.BOUNCER,
            constants.BOUNCER_BULLET_SPEED, constants.BOUNCER_PROJECTILE_DAMAGE, id, agentId)
    }
}

module.exports = BouncerProjectile;

File: core/src/com/mygdx/game/model/projectile/BouncerProjectile.kt
package com.mygdx.game.model.projectile

import com.badlogic.gdx.graphics.Texture
import com.mygdx.game.settings.BOUNCER_BULLET_SPEED
import com.mygdx.game.settings.BOUNCER_PROJECTILE_WIDTH

class BouncerProjectile(
        x: Float = 0f,
        y: Float = 0f,
        xSpeed: Float = 0f,
        ySpeed: Float = 0f,
        texture: Texture) : Projectile(
        x,
        y,
        xSpeed,
        ySpeed,
        texture,
        BOUNCER_PROJECTILE_WIDTH / 2,
        BOUNCER_BULLET_SPEED)